integrations/hadoop/documentation.yaml

```diff
@@ -0,0 +1,143 @@
+exporter_type: sidecar
+app_name_short: Apache Hadoop
+app_name: Apache {{app_name_short}}
+app_site_name: Apache Hadoop
+app_site_url: https://hadoop.apache.org/
+exporter_name: the JMX exporter
+exporter_pkg_name: jmx-exporter
+exporter_repo_url: https://github.com/bitnami/containers/tree/main/bitnami/jmx-exporter
+dashboard_available: true
+minimum_exporter_version: "0.17.0"
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+config_mods: |
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: hadoop-exporter
+  data:
+    config.yaml: |
+      hostPort: localhost:1026
+      lowercaseOutputName: true
+      lowercaseOutputLabelNames: true
+  ---
+  apiVersion: apps/v1
+  kind: StatefulSet
+  metadata:
+    name: hadoop-hdfs
+  spec:
+    serviceName: hadoop-hdfs
+    selector:
+      matchLabels:
+  +     app.kubernetes.io/name: hadoop
+    template:
+      metadata:
+        labels:
+  +       app.kubernetes.io/name: hadoop
+      spec:
+        containers:
+        - name: hadoop-hdfs
+          image: "farberg/apache-hadoop:3.3.2"
+  +       env:
+  +         - name: HDFS_NAMENODE_OPTS
+  +           value: "-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.port=1026"
+          command:
+          - "/bin/bash"
+          - "/tmp/hadoop-config/bootstrap.sh"
```

Comment:
When I tried following these instructions to set up the exporter for hadoop, I ran into an error because the hadoop configuration is incomplete. It looks like we need to define the bootstrap.sh script for hadoop and mount it as a hadoop-config volume.

We can either:
1. Add the complete config that will set up hadoop and then set up the exporter to export metrics
2. If the above is too much config and out of scope for the document, we should add a "..." to the hadoop-hdfs config and add an additional_install_info field similar to how we have for Redis, making it clear that we expect them to have a working hadoop application, or pointing to a tutorial if they don't.

Reply:
Apologies, the way Hadoop was deployed on our end was using this [helm chart](https://artifacthub.io/packages/helm/apache-hadoop-helm/hadoop). We deployed it, generated a YAML file, then updated that to include the exporter. 

I think we would want to specify that the helm chart was used and is necessary for the copy/pasting to work appropriately. If a user downloads the templates and updates them accordingly, that should work. 

I've added an `additional_install_info` field and made it clear that these instructions are based on modifying the templates from that helm chart.

Reply:
Got it - I was able to install it using the helm chart and complete the set up.